compiler/src/iree/compiler/Codegen/Common/TileSizeSelection.cpp

```diff
@@ -29,7 +31,7 @@ TilingConfig::TilingConfig(IREE::Codegen::LoweringConfigAttr lc)
   //   5. [[distribution], [cache-parallel], [cache-reduction],
   //       [vector-common-parallel], [vector-reduction],
   //       [vector-inner-parallel]]
-  int numTileLevels = loweringConfig.getTilingLevels().size();
+  int numTileLevels = getNumTilingLevels();
```

Comment:
```suggestion
  unsigned numTileLevels = getNumTilingLevels();
```
Nit: Use the actual return type of the function here?